order_analysis.rb

```diff
@@ -3,17 +3,20 @@
 
 class OrderAnalysis
   CSV_DELIMITER = ";"
-
+  def initialize
+    @customers = []
+  end
   def analyze
     analyze_customers
+    analyze_orders
   end
-
   def analyze_customers
     customers_csv = File.read("customers.csv")
```

Comment:
In most cases, you shouldn‘t hardcode file names. It‘s okey in a context of an exercise tough :)